etc/kayobe/ansible/deploy-radosgw-usage-exporter.yml

```diff
@@ -0,0 +1,152 @@
+---
+- name: Deploy RADOS gateway usage exporter
+  hosts: monitoring
+  gather_facts: false
+  tags: radosgw_usage_exporter
+  vars:
+    venv: "{{ virtualenv_path }}/openstack"
+  tasks:
```

Comment:
Could you add the `when: stackhpc_enable_radosgw_usage_exporter` to the play rather than each task individually? If not, then can you add the tasks to a block and add it there?